axelor-crm/src/main/resources/data-init/input/crm_ticketCategory.csv

```diff
@@ -1,8 +1 @@
 importId;name
```

Comment:
CRM ticket doesn"t exist anymore, so we can remove this file on crm module and move it to helpdesk TicketType.